templates/zerver/help/include/disable-video-call-button.md

```diff
@@ -0,0 +1,4 @@
+You can also disable the video call button for your organization by
+setting the provider to "None".
+
+![](/static/images/integrations/disable-video-call-button.png)
```

Comment:
I would consider having this just be a section on the "start-a-call" page, since it'd be such a tiny article otherwise.

Also, https://zulip.readthedocs.io/en/latest/documentation/user.html is useful reference, but basically we try to avoid screenshots wherever possible because they are expensive to maintain.

Reply:
I actually did refer [https://zulip.readthedocs.io/en/latest/documentation/user.html](https://zulip.readthedocs.io/en/latest/documentation/user.html), but as `/integration` doc was becoming too short so I thought of this approach to also add steps about disabling the video-call button and included image so that it covers some space and looks decent, I had similar doubts about adding image but then I thought I would get appropriate instructions during review.